.github/workflows/test.yml

```diff
@@ -0,0 +1,21 @@
+name: pull-request
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+      - uses: actions/checkout@v2
+
+      - name: Set up node 
+        uses: actions/setup-node@v1
```

Comment:
We don't need to set up node, it's already included in `ubuntu-latest`.
```suggestion
```

See https://github.com/actions/runner-images/blob/ubuntu20/20220807.1/images/linux/Ubuntu2004-Readme.md#language-and-runtime.